src/components/policy/test/policy_manager_impl_test.cc

```diff
@@ -86,7 +89,44 @@ class PolicyManagerImplTest : public ::testing::Test {
     delete listener;
   }
 
-  ::testing::AssertionResult IsValid(const policy_table::Table &table) {
+  ::testing::AssertionResult IsValid(const policy_table::Table& table) {
+    if (table.is_valid()) {
+      return ::testing::AssertionSuccess();
+    } else {
+      ::rpc::ValidationReport report(" - table");
+      table.ReportErrors(&report);
+      return ::testing::AssertionFailure() << ::rpc::PrettyFormat(report);
+    }
+  }
+};
+
+class PolicyManagerImplTest2 : public ::testing::Test {
```

Comment:
@orhan-mehmedov Please at least add comments why do you need PolicyManagerImplTest2?